distro_data/Linux_Software_List.json

```diff
@@ -0,0 +1,737 @@
+[
```

Comment:
What is the source for this data for Linux? It seems pretty generic, not comprehensive, and isn't distro-specific, so I'm wondering about the value, and how it would show up in the search (what checkbox you'd use to get to this data)